libunwind/include/libunwind.h

```diff
@@ -679,10 +679,6 @@ enum {
   UNW_ARM64_CLR = 228,
   UNW_ARM64_C31 = 229,
   UNW_ARM64_CSP = 229,
-  // Use 240 for ECSP (executive stack pointer). ECSP is not a real DWARF
-  // register, but we need it to implement c18n-aware unwinding. We pick 240
-  // because it is far enough away from the range of reserved registers on Arm.
-  UNW_ARM64_ECSP = 240,
```

Comment:
I guess in an earlier iteration of the patch we really did need it to be a DWARF register?..